plugins/modules/packaging/os/xbps.py

```diff
@@ -50,21 +50,42 @@
             - Whether or not to upgrade whole system
         type: bool
         default: 'no'
+    upgrade_xbps:
+        description:
+            - Whether or not to upgrade the xbps package when necessary.
+              Before installing new packages,
+              xbps requires the user to update the xbps package itself.
+              Thus when this option is set to C(no),
+              upgrades and installations will fail when xbps is not up to date.
+        type: bool
+        default: 'yes'
+        version_added: '2.10'
```

Comment:
If `version_added` will be used in the future, it will refer to the collection version.
```suggestion
```

Reply:
done